nurseconnect_registration/urls.py

```diff
@@ -18,10 +18,12 @@
 from django_prometheus.exports import ExportToDjangoView as metrics
 
 from nurseconnect_registration.decorators import internal_only
+from nurseconnect_registration.views import TermsAndConditionsView
 
 urlpatterns = [
     path("", include(("registrations.urls", "registrations"))),
     path("admin/", admin.site.urls),
     path("metrics", internal_only(metrics), name="metrics"),
     path("health/", include(("watchman.urls", "health"))),
+    path("terms_and_conditions/", TermsAndConditionsView.as_view()),
```

Comment:
I think the terms and conditions URL, view, etc, can all be apart of the `registrations` app, since these are the terms and conditions for the registration.

Reply:
ahh I see, okay will put it under that